test/fixtures/basic/pages/random-catchall/[...id].vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+    <NuxtLink to="/" prefetched-class="prefetched">
+      Home
+    </NuxtLink>
+    <NuxtLink to="/random-catchall/a/" prefetched-class="prefetched">
+      Random (A)
+    </NuxtLink>
+    <NuxtLink to="/random-catchall/b/" prefetched-class="prefetched">
+      Random (B)
+    </NuxtLink>
+    <NuxtLink to="/random-catchall/c/" prefetched-class="prefetched">
+      Random (C)
+    </NuxtLink>
+    <ServerOnlyComponent />
+    <br>
+
+    Random: {{ random }}
+
+    Random: (global) {{ globalRandom }}
+
+    Random page: <b>{{ route.params.id }}</b><br>
+
+    Here are some random numbers for you:
+
+    <ul>
+      <li v-for="n in randomNumbers" :key="n">
+        {{ n }}
+      </li>
+    </ul>
+    <button @click="() => refresh()">
+      Give me another set
+    </button>
+  </div>
+</template>
+
+<script setup lang="ts">
+const route = useRoute()
+
+const pageKey = 'rand_' + route.params.id
```

Comment:
The issue here is that the key is either `rand_a,` or `rand_a` depending on whether or not there is a trailing slash. So this will always mismatch.

Reply:
Whoops, you are right 🙈 Will fix the test 👍🏻